src/effects/index.ts

```diff
@@ -85,5 +86,15 @@ export const CHAT_EFFECTS: Array<Effect<{ [key: string]: any }>> = [
             gravity: 0.01,
         },
     } as Effect<SpaceInvadersOptions>,
+    {
+        emojis: ["💝"],
+        msgType: "io.element.effect.hearts",
+        command: "hearts",
+        description: () => _td("Sends the given message with hearts"),
+        fallbackMessage: () => _t("sends hearts") + " 💞",
```

Comment:
For consistency with the trigger emoji

```suggestion
        fallbackMessage: () => _t("sends hearts") + " 💝",
```